keylime/config.py

```diff
@@ -19,9 +19,7 @@ def convert(data: Any) -> Any:
     if isinstance(data, bytes):
         return data.decode()
     if isinstance(data, dict):
-        return dict(iter(map(convert, data.items())))  # pyright: ignore
-    if isinstance(data, tuple):
-        return tuple(map(convert, data))
+        return dict(cast(Iterator[Any], iter(map(convert, data.items()))))
```

Comment:
Did you drop the `tuple` case on purpose?

Reply:
No, this was deleted by accident. Thanks.

Reply:
Do we want to tolerate the cast here (not sure whether this is a bug or feature now of pyright) or go with ignoring the line? Or try yet something else?

Reply:
seems indeed as a bug to me? would make sense to keep it and create a bug report in pyright? Depending on the outcome we revert this, or address the issue differently.

Reply:
It must be a bug related to running pyright under `tox`. When I revert the changes then I get no errors with this command line:

```
# pyright keylime/config.py
WARNING: there is a new pyright version available (v1.1.281 -> v1.1.287).
Please install the new version or set PYRIGHT_PYTHON_FORCE_VERSION to `latest`
[...]
0 errors, 0 warnings, 0 informations
Completed in 1.266sec
```

With `tox -epyright` the reported errors appear. Hm.

Reply:
[delete previous posting. picked up old pyright version]

Reply:
https://github.com/microsoft/pyright/issues/4428

Reply:
Considering the response I got in the pyright project and the fact that they release a new version ever week (Monday), this problem will go away next week (latest) since it's a bug in pyright.

Reply:
An anticipation of the new pyright release I am closing this PR now.

Reply:
Thanks for creating the upstream bug. This was really helpful!